yarn.lock

```diff
@@ -1545,6 +1545,32 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.14.0.tgz#1483caa1dd4e7097ac48a5872e93d7f896e8402c"
   integrity sha512-WNNVTMpRir6a2Voa0yu3BZJmy2w7fofrokL+nyFQM40bACtX68KDRDE96HA5W5+4q8rEaNSXV3DzSzLXm6ekfA==
 
+"@fluentui-react-native/experimental-shadow@0.2.12":
```

Comment:
If this is showing up in the lock file, I think we're using an older version of shadow than what's in the repo?

Reply:
Updated